sdks/java/extensions/euphoria/euphoria-core/src/main/java/org/apache/beam/sdk/extensions/euphoria/core/client/io/ListDataSink.java

```diff
@@ -40,7 +41,10 @@
       Collections.synchronizedMap(new WeakHashMap<>());
 
   private final int sinkId = System.identityHashCode(this);
+
+  @SuppressFBWarnings("SE_BAD_FIELD")
```

Comment:
is it because, its used only for testing?

Reply:
This is in fact very similar to the `BeamFlow` case.